src/pe/import.rs

```diff
@@ -263,18 +263,29 @@ impl<'a> SyntheticImportDirectoryEntry<'a> {
             }
         };
 
-        let import_address_table_offset = &mut utils::find_offset(
-            import_directory_entry.import_address_table_rva as usize,
-            sections,
-            file_alignment,
-            opts,
-        )
-        .ok_or_else(|| {
-            error::Error::Malformed(format!(
-                "Cannot map import_address_table_rva {:#x} into offset for {}",
-                import_directory_entry.import_address_table_rva, name
-            ))
-        })?;
+        let rva = match import_directory_entry.import_address_table_rva.is_zero() {
+            true => import_directory_entry.import_lookup_table_rva,
+            false => import_directory_entry.import_address_table_rva,
+        };
+
+        let import_address_table_offset =
+            &mut utils::find_offset(rva as usize, sections, file_alignment, opts).ok_or_else(
+                || {
+                    error::Error::Malformed(
+                        if import_directory_entry.import_address_table_rva.is_zero() {
+                            format!(
+                                "Cannot map import_lookup_table_rva {:#x} into offset for {}",
+                                rva, name
+                            )
+                        } else {
+                            format!(
+                                "Cannot map import_address_table_rva {:#x} into offset for {}",
+                                rva, name
+                            )
+                        },
+                    )
```

Comment:
this can be taken to extreme, but i definitely think this is better and less verbose:
```rust
let target = if import_directory_entry.import_address_table_rva.is_zero() { "import_lookup_table_rva" } else { "import_address_table_rva" };
error::Error::Malformed(format!("Cannot map {target} {rva:#x} into offset for {name}")
```

you could also have the rva branch above return a tuple of (rva, target) to use later on if there's an error, but this version probably better/cleaner/more local